src/exeptions/apiError.js

```diff
@@ -0,0 +1,34 @@
+class ApiError extends Error {
+  constructor({ message, status, errors = {} }) {
+    super(message);
+
+    this.status = status;
+    this.errors = errors;
+  }
+
+  static badRequest(message, errors) {
+    return new ApiError({
+      message,
+      errors,
+      status: 400,
+    });
+  }
+
+  static unauthorized(errors) {
```

Comment:
Consider providing a default empty object for the `errors` parameter in the `unauthorized` method to maintain consistency with other static methods.